<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_equip_1</class>
 <widget class="QWidget" name="page_equip_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>36</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ExtendedComboBox" name="hero_select">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>15</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filter_btn">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 사도 목록 필터 </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>36</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="go_left_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="rank_table">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="go_right_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="font">
      <font>
       <family>ONE Mobile POP</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_mode">
       <property name="font">
        <font>
         <family>ONE Mobile POP</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>모드</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_cur_mode">
          <property name="font">
           <font>
            <family>ONE Mobile POP</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>착용 장비 기록</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radio_goal_mode">
          <property name="font">
           <font>
            <family>ONE Mobile POP</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>목표 설정</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <property name="font">
           <font>
            <family>ONE Mobile POP</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>ONE Mobile POP</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>현재 목표 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cur_goal">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>ONE Mobile POP</family>
               <pointsize>13</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="goal_setting_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>ONE Mobile POP</family>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string> 목표 수정/삭제 </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_setAll">
         <property name="font">
          <font>
           <family>ONE Mobile POP</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="title">
          <string>일괄 설정하기</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>ONE Mobile POP</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>ONE Mobile POP</family>
                   <pointsize>13</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>현재 모드에서 전체 사도</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_set_goal">
                 <property name="font">
                  <font>
                   <family>ONE Mobile POP</family>
                   <pointsize>13</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>ONE Mobile POP</family>
                   <pointsize>13</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>랭크까지</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="set_check_btn">
                 <property name="font">
                  <font>
                   <family>ONE Mobile POP</family>
                   <pointsize>13</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>체크</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer">
               <property name="font">
                <font>
                 <family>ONE Mobile POP</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>93</width>
                 <height>38</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="set_goal_w_stat_btn">
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>ONE Mobile POP</family>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>스탯 별 랭크작 목표 일괄 설정</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="font">
                <font>
                 <family>ONE Mobile POP</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>38</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/wrapper/misc.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
